static/app/components/workflowEngine/gridCell/connectionCell.tsx

```diff
@@ -10,28 +10,35 @@ import {tn} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {AvatarProject} from 'sentry/types/project';
 
-export type Monitor = {
-  id: string;
+export type Item = {
+  link: string;
   name: string;
   project: AvatarProject;
   description?: string;
 };
 
-type MonitorsCellProps = {
-  monitors: Monitor[];
+export type ConnectionCellProps = {
+  items: Item[];
+  type: 'monitor' | 'automation';
 };
 
-export function MonitorsCell({monitors}: MonitorsCellProps) {
-  if (monitors.length === 0) {
+const text = (type: 'monitor' | 'automation', length: number) => {
+  if (type === 'monitor') {
+    return tn('%s monitor', '%s monitors', length);
+  }
+  return tn('%s automation', '%s automations', length);
+};
```

Comment:
Instead of a `type` prop that is limited to `monitor | automation`, what if this component accepted a `renderSuffix` function? That would be slightly more composable and avoid hardcoding this logic inline.